default.properties

```diff
@@ -37,6 +37,16 @@ useLegacyHeader=false
 # default /header/
 headerUrl=/header/
 
+# Stylesheet used to override default colors of header
+# More design can be set by overriding default classes & styles
+# Default empty string
+#georchestraStylesheet=http://localhost:3000/tmp.css
+
+# Logo URL
+# Used to set header's logo.
+logoUrl=https://www.georchestra.org/public/georchestra-logo.svg
```

Comment:
As already mentioned, it would be nice to be able to point to a local resource here in the future.